test/js-api/js-promise-integration/js-promise-integration.any.js

```diff
@@ -149,19 +152,21 @@ test(() => {
       {suspending: 'first'});
   let instance = builder.instantiate({m: {import: wasm_js_import}});
   let wrapped_export = ToPromising(instance.exports.test);
-  assert_equals(42, wrapped_export());
+  wrapped_export();
+  // The global was updated synchronously.
```

Comment:
Is it correct that with amends in PR #16 this global will not be updated synchronously.

Reply:
That is correct. But note that this PR has some downsides and it is not clear at this point whether it will make it into the proposal. Ping @fgmccabe who has been thinking about this issue recently.